### ISSUE TITLE
meshcat backgrounds are broken for orthographic cameras

### ISSUE THREAD

**Issue by RussTedrake** [RobotLocomotion/drake#19959]
### What happened?

```
meshcat = StartMeshcat()
meshcat.Set2dRenderMode(X_WC=RigidTransform(RollPitchYaw(-np.pi / 2, 0, 0),
                                            [0, 0, 1]),
                        xmin=-2,
                        xmax=2,
                        ymin=-2,
                        ymax=2)
meshcat.SetProperty("/Background", "visible", True)
```
produces
<img width="723" alt="image" src="https://github.com/RobotLocomotion/drake/assets/6442292/bc2412d7-1ec9-4685-b323-b78566299b62">

Expected: the entire background should have the background color.

### What operating system are you using?

mac ventura, chrome

### What installation option are you using?

build from source

**Comment by SeanCurtis-TRI** [comment]
Root cause solved in: https://github.com/rdeits/meshcat/pull/150.

**Comment by SeanCurtis-TRI** [comment]
Resolved when meshcat was updated in #20000.
